# # Set fixed waypoint position
# Simple (for testing)
plan_wp_x: [5]
plan_wp_y: [0]
plan_wp_z: [-0.5]
# Going to last waypoint fast
# plan_wp_x: [5, 7, 24]
# plan_wp_y: [0, 0, 0]
# plan_wp_z: [-0.5, -25, -32]
# Going to last waypoint slower
# plan_wp_x: [5, 9, 7, 7, 7, 11, 14, 17, 24]
# plan_wp_y: [0, 0, 0, 0, 0, 0, 0, 0, 0]
# plan_wp_z: [-0.5, -15, -1, -10, -25, -27, -28, -29, -32]